shared/team-building/user-result.tsx

```diff
@@ -305,36 +294,37 @@ const ActionButton = (props: {inTeam: boolean; onAdd: () => void; onRemove: () =
   )
 }
 
-const AddButton = () => <Kb.Icon type="iconfont-circle" fontSize={22} color={Styles.globalColors.black_10} />
+const AddButton = () => (
+  <Kb.Icon type="iconfont-circle" fontSize={actionButtonSize} color={Styles.globalColors.black_10} />
+)
 
 const AlreadyAddedIconButton = () => (
-  <Kb.Icon type="iconfont-success" fontSize={22} color={Styles.globalColors.blue} />
+  <Kb.Icon type="iconfont-success" fontSize={actionButtonSize} color={Styles.globalColors.blue} />
 )
 
-const ActionButtonSize = 40
-export const userResultHeight = Styles.globalMargins.xlarge
+export const userResultHeight = Styles.isMobile ? Styles.globalMargins.xlarge : 48
 const styles = Styles.styleSheetCreate(() => ({
   actionButton: Styles.platformStyles({
     common: {
-      height: ActionButtonSize,
       marginLeft: Styles.globalMargins.tiny,
-      width: ActionButtonSize,
+    },
+    isElectron: {
+      height: Styles.globalMargins.small,
+      width: Styles.globalMargins.small,
     },
     isMobile: {
-      marginRight: 0,
+      height: Styles.globalMargins.large,
+      marginRight: Styles.globalMargins.tiny,
+      width: Styles.globalMargins.large,
     },
   }),
-  addToTeamIcon: {
-    ...Styles.globalStyles.rounded,
-    height: ActionButtonSize,
-    width: ActionButtonSize,
-  },
   bottomRowContainer: {
     alignItems: 'baseline',
     flexWrap: 'nowrap',
     overflow: 'hidden',
   },
   bottomRowScrollContainer: {
+    alignItems: 'baseline',
```

Comment:
This is so that the username and prettyName will not have a 2px baseline offset.